src/applications/vaos/utils/appointment.js

```diff
@@ -133,12 +133,13 @@ function getRequestDateOptions(appt) {
 }
 
 export function getPastAppointmentDateRangeOptions(today = moment()) {
+  const startOfToday = today.startOf('day');
```

Comment:
Having date times down to the second in the query params for fetching appointments was really hard to test and I think also potentially confusing to users. So this sets everything to the start of the day instead.